include/dfu/dfu_target.h

```diff
@@ -0,0 +1,93 @@
+/*
+ * Copyright (c) 2019 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+ */
+
+#ifndef DFU_TARGET_H__
+#define DFU_TARGET_H__
+
+/**
+ * @defgroup dfu_target DFU Target
+ * @brief DFU Target
+ *
+ * @{
+ */
+
+#include <zephyr/types.h>
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#define DFU_TARGET_IMAGE_MCUBOOT 1
+#define DFU_TARGET_IMAGE_MODEM_DELTA 2
```

Comment:
Naming suggestion: `DFU_TARGET_IMG_TYPE_*`

To follow function naming `dfu_target_img_type()`